apps/catalog/catalog/src/app/dashboard/sales/shell/shell.component.html

```diff
@@ -0,0 +1,42 @@
+<ng-container *ngIf="sale$ | async as sale; else loading">
+  <header fxLayout="column" fxLayoutAlign="start start">
+    <div fxLayout fxlayoutAlign="space-between center">
+      <a mat-icon-button routerLink="/c/o/dashboard/sales" color="primary">
+        <mat-icon svgIcon="arrow_back"></mat-icon>
+      </a>
+
+      <a mat-button routerLink="/c/o/dashboard/standard-terms" color="primary">
+        <mat-icon svgIcon="file"></mat-icon>
+        <span>See Standard Terms</span>
+      </a>
+    </div>
+    <bf-tag>{{sale.status | toLabel: 'contractStatus'}}</bf-tag>
+    <ng-container *ngIf="sale.sellerId === centralOrgId.catalog; else externalSale">
+      <ng-container *ngIf="sale.buyerId | getOrg | async as org">
+        <div fxLayout fxLayoutAlign="start center" fxLayoutGap="16px">
+          <img height="100" width="100" [ref]="org?.logo" asset="empty_organization.svg" type="logo"
+            alt="Organization Logo">
+          <h1>{{ org | orgName }}</h1>
+        </div>
+      </ng-container>
+    </ng-container>
+    <ng-template #externalSale>
+      <div fxLayout fxLayoutAlign="start center" fxLayoutGap="16px">
+        <img height="100" width="100" asset="empty_organization.svg" type="logo" alt="Organization Logo">
+        <h1>External</h1>
+      </div>
+    </ng-template>
+    <div fxLayout fxLayoutAlign="space-between center" fxLayoutGap="16px">
+      <span>Ref {{ sale.offerId }} / Offer created: {{sale._meta.createdAt | date: 'MM/dd/yyyy'}}</span>
+      <button mat-button color="primary" (click)="openIntercom()">
+        <mat-icon svgIcon="comment"></mat-icon>
+        <span>Contact Archipel Content Team</span>
+      </button>
+    </div>
+
+  </header>
+</ng-container>
+<sale-view></sale-view>
```

Comment:
this should be a `router-outlet` so we can manage the pages with a Router
```suggestion
<router-outlet></router-outlet>
```